staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -27,6 +27,11 @@ const (
 	// More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
 	labelPrefix = "batch.kubernetes.io/"
 
+	// CronJobScheduledTimestampAnnotation is the scheduled timestamp annotation for the Job.
+	// It records the original/expected scheduled timestamp for the running job, represented in RFC3339.
+	// The CronJob controller adds this annotation if the CronJobsScheduledAnnotation feature gate (beta in 1.28) is enabled.
+	CronJobScheduledTimestampAnnotation = labelPrefix + "cronjob-scheduled-timestamp"
```

Comment:
this looks good.. open a PR to the dev-1.28 branch of the website to add this doc to https://kubernetes.io/docs/reference/labels-annotations-taints/ as well

Reply:
PR is ready already. https://github.com/kubernetes/website/pull/41857